### PR TITLE
Updates to refactored AWS cloud prepare code

### PR DESCRIPTION
**Pull request by tpantelis**
Minor miscellaneous changes that weren't noted on the original review.

### CODE REVIEW

**Review by Jaanki — COMMENTED** [review]
While you are at it, can you please rename [`var aws`](https://github.com/submariner-io/submariner-operator/blob/devel/pkg/subctl/cmd/cloud/prepare/aws.go#L32)  to `awsConfig` to match with other platforms.

**Review by skitt — APPROVED** [review]

**Comment by tpantelis** [comment]
> While you are at it, can you please rename [`var aws`](https://github.com/submariner-io/submariner-operator/blob/devel/pkg/subctl/cmd/cloud/prepare/aws.go#L32) to `awsConfig` to match with other platforms.

Where is this? Can you comment on the actual line?

File: pkg/subctl/cmd/cloud/prepare/aws.go [hunk 1]
```diff
@@ -29,15 +29,15 @@ import (
...
-
 const (
 	infraIDFlag = "infra-id"
 	regionFlag  = "region"
 )
 
 // NewCommand returns a new cobra.Command used to prepare a cloud infrastructure.
-func newAWSPrepareCommand(restConfigProducer *restconfig.Producer, ports cloud.Ports) *cobra.Command {
+func newAWSPrepareCommand(restConfigProducer *restconfig.Producer, ports *cloud.Ports) *cobra.Command {
+	awsConfig := &cloudaws.Config{}
```
Comment (tpantelis) [inline pkg/subctl/cmd/cloud/prepare/aws.go:39]: I made this local.

File: pkg/subctl/cmd/cloud/cleanup/aws.go [hunk 2]
```diff
@@ -23,13 +23,9 @@ import (
 	"github.com/submariner-io/submariner-operator/internal/cli"
 	"github.com/submariner-io/submariner-operator/internal/exit"
 	"github.com/submariner-io/submariner-operator/internal/restconfig"
-	cloudaws "github.com/submariner-io/submariner-operator/pkg/cloud/aws"
 	"github.com/submariner-io/submariner-operator/pkg/cloud/cleanup"
-	"github.com/submariner-io/submariner-operator/pkg/subctl/cmd/cloud/aws"
 )
 
-var config cloudaws.Config
```
Comment (tpantelis) [inline pkg/subctl/cmd/cloud/cleanup/aws.go:31]: This is unused for cleanup.

Comment (Jaanki) [reply pkg/subctl/cmd/cloud/cleanup/aws.go:31]: Not really. As discussed over slack, these flags are needed for `cleanup`. See changes I made for RHOS https://github.com/submariner-io/submariner-operator/pull/2048/files#diff-3561b3b903372de9ecaaaf82c597d35ca4a3a9a1dbefe02273b5bcd8519a686d

**Comment by Jaanki** [comment]
> > While you are at it, can you please rename [`var aws`](https://github.com/submariner-io/submariner-operator/blob/devel/pkg/subctl/cmd/cloud/prepare/aws.go#L32) to `awsConfig` to match with other platforms.
> 
> Where is this? Can you comment on the actual line?

> https://github.com/submariner-io/submariner-operator/pull/2048/files#diff-3561b3b903372de9ecaaaf82c597d35ca4a3a9a1dbefe02273b5bcd8519a686dR33 

I'm confused - that's RHOS code. You said there's a variable named `aws` that needs to be renamed. I don't see that anywhere - can you point it out?

**Review by Jaanki — CHANGES_REQUESTED** [review]
Also appropriate changes needed for `cleanup` to use AWS flags.

File: pkg/subctl/cmd/cloud/cleanup/aws.go (hunk 2 above)
Comment (tpantelis) [reply pkg/subctl/cmd/cloud/cleanup/aws.go:31]: yeah I'm fixing that in another PR.

**Comment by tpantelis** [comment]
I rolled these changes into https://github.com/submariner-io/submariner-operator/pull/2053.

**Pull request closed without merging**